docker/pytorch/mivisionx-with-pytorch-with-mesa.dockerfile

```diff
@@ -0,0 +1,59 @@
+ARG PYTORCH_VERSION=latest
+FROM rocm/pytorch:${PYTORCH_VERSION}
+
+ENV MIVISIONX_DEPS_ROOT=/opt/mivisionx-deps
```

Comment:
@kiritigowda can you make the same changes you did with path on the other dockers for mivisionx-deps to this file also?